refinery/factory_boy/utils.py

```diff
@@ -1,11 +1,15 @@
-from datetime import datetime
 import uuid as uuid_builtin
 
-from core.models import Analysis, DataSet, Node
+from core.models import INPUT_CONNECTION, OUTPUT_CONNECTION, Analysis
+from data_set_manager.models import Node
 from factory_boy.django_model_factories import (AnalysisFactory,
+                                                AnalysisNodeConnectionFactory,
```

Comment:
Use the whole allowed width of the line

Reply:
`isort` formats things this way. We'd have to change its configuration if want wanted something different

Reply:
OK then we should change its config.